Source/WebInspectorUI/UserInterface/Views/LogContentView.js

```diff
@@ -1333,6 +1323,25 @@ WI.LogContentView = class LogContentView extends WI.ContentView
         if (this.element.firstChild !== this._hiddenMessagesBannerElement)
             this.element.insertAdjacentElement("afterbegin", this._hiddenMessagesBannerElement);
     }
+
+    _showConditionallyVisibleScopeBarItemsIfNeeded(newMessage) {
+        for (let scope of WI.LogContentView.ConditionallyVisibleScopes) {
```

Comment:
it would probably be more efficient to do this
```js
_showConditionallyVisibleScopeBarItemsIfNeeded(newMessage)
{
    let scope = this._scopeFromMessageLevel(newMessage.level);

    if (!WI.LogContentView.ConditionallyVisibleScopes.includes(scope))
        return;

    this._scopeBar.item(scope).hidden = false;
}
```